core/domain/exp_jobs_one_off.py

```diff
@@ -791,3 +791,49 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class TranslatorToVoiceArtistOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for migrating translator_ids to voice_artist_ids."""
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationRightsModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        translator_ids = item.translator_ids
+        commit_message = 'Migrate from translator to voice artist'
+        commit_cmds = [{
+            'cmd': 'change_role',
+            'assignee_id': translator_id,
+            'old_role': 'translator',
```

Comment:
Should use rights_manager.ROLE_TRANSLATOR, ditto for new_role below. Always use named constants where possible. (That way, if a constant ever needs to be changed, a Ctrl-F will find it.)

Reply:
ROLE_TRANSLATOR was removed in rights_manager already. Should I add a deprecated value then? (seems a little weird to me tho)

Reply:
Oh, that's a good point. OK to keep that as a string, then, but change new_role accordingly.

Reply:
Done.